### ISSUE TITLE
Build METEOR_SETTINGS from env

### ISSUE THREAD

**Issue by kaihendry** [unee-t/frontend#55]
So the [build deploy stage was breaking](https://travis-ci.org/unee-t/frontend/builds/324476057) due to the JSON. Stringified still seems to make YAML croak.

But looking at:

```
       - CLOUDINARY_URL=https://api.cloudinary.com/v1_1/unee-t-staging/image/upload
       - CLOUDINARY_PRESET=ABC
       - COMMIT=a876254
       - METEOR_SETTINGS="{ \"public\": { \"CLOUDINARY_URL\": \"https://api.cloudinary.com/v1_1/unee-t-dev/image/upload\", \"CLOUDINARY_PRESET\": \"XYZ\"
} }"

```

Why do we have 2x CLOUDINARY_URL & 2x CLOUDINARY_PRESET that are different!?

Can we please keep to just KEY=VALUE type environment variables? :grin:

**Comment by franck-boullier** [comment]
error message for PR53:
![image](https://user-images.githubusercontent.com/31331637/34592153-15d5cb20-f1fd-11e7-8172-dcca7b2b4737.png)

**Comment by nbiton** [comment]
It's actually unrelated :sweat_smile:

**Comment by nbiton** [comment]
METEOR_SETTINGS is special env var, that's treated in a different way by Meteor. The "public" part is shipped directly to the client, so it is available on initial page load. We can't use other env vars instead.

**Comment by kaihendry** [comment]
As discussed on appear.in, you will build it. The embedding JSON approach seems too fragile.

ecs-cli seems to realise it's a map when parsing it and croaks for e.g.

**Comment by franck-boullier** [comment]
@nbiton For the records, dev.case.unee-t.com is still giving me the 'ever loading page error'
![image](https://user-images.githubusercontent.com/31331637/34596482-06abb966-f21b-11e7-9b3b-b46a643c9f84.png)

(the heroku interface works OK)

**Comment by nbiton** [comment]
I started working on PR to do what @hendry described above. Once merged, it should fix the issue on dev

**Comment by nbiton** [comment]
Resolved by #57
